main.tf

```diff
@@ -31,6 +31,7 @@ resource "google_compute_global_address" "static_ip" {
 }
```

Comment:
I guess if we're letting GKE manage the LB then is the static IP necessary?

Reply:
Good call